projects/common/helpers/deburr.ts

```diff
@@ -0,0 +1,2 @@
+// Normalize accented characters in a string
+export const deburr = (str: string): string => str.normalize("NFD").replace(/[\u0300-\u036f]/g, "");
```

Comment:
Can you add some unit tests to confirm that this works for our cases?

Reply:
Also, does this work with IE11?

Reply:
Replaced with function that doesn't use String.prototype.normalize and added unit tests